ignored_ci_repos.yml

```diff
@@ -1,3 +1,7 @@
+- ckanext-datagovuk
+- ckanext-spatial
+- datagovuk_find
+- datagovuk_publish
```

Comment:
Can you explain in the commit message:

1) Why do these repos need to be ignored?
2) Why weren't we ignoring them already?

Thanks!

Reply:
I've spoken to Chris offline and we've decided to drop this commit.